doc/fr/developper/writestreamconnector.rst

```diff
@@ -0,0 +1,586 @@
+  mer. 28 mars 2018 14:27:35 CEST: INFO: enabled => true
+  mer. 28 mars 2018 14:27:35 CEST: INFO: host_id => 102
+  mer. 28 mars 2018 14:27:35 CEST: INFO: last_time_ok => 1522240053
+  mer. 28 mars 2018 14:27:35 CEST: INFO: state => 0
+  mer. 28 mars 2018 14:27:35 CEST: INFO: last_update => 1522240054
+  mer. 28 mars 2018 14:27:35 CEST: INFO: last_check => 1522240053
+  mer. 28 mars 2018 14:27:35 CEST: INFO: execution_time => 0.005025
+  mer. 28 mars 2018 14:27:35 CEST: INFO: acknowledged => false
+  mer. 28 mars 2018 14:27:35 CEST: INFO: service_id => 778
+  mer. 28 mars 2018 14:27:35 CEST: INFO: active_checks => true
+  mer. 28 mars 2018 14:27:35 CEST: INFO: notify => false
+  mer. 28 mars 2018 14:27:35 CEST: INFO: max_check_attempts => 3
+  mer. 28 mars 2018 14:27:35 CEST: INFO: obsess_over_service => true
+  mer. 28 mars 2018 14:27:35 CEST: INFO: check_type => 0
+  mer. 28 mars 2018 14:27:35 CEST: INFO: last_hard_state_change => 1522165654
+  mer. 28 mars 2018 14:27:35 CEST: INFO: category => 1
+  mer. 28 mars 2018 14:27:35 CEST: INFO: perfdata => used=41986296644o;48103633715;54116587930;0;60129542144 size=60129542144o
+  mer. 28 mars 2018 14:27:35 CEST: INFO: check_interval => 5
+  mer. 28 mars 2018 14:27:35 CEST: INFO: output => Disk /var - used : 39.10 Go - size : 56.00 Go - percent : 69 %
+  mer. 28 mars 2018 14:27:35 CEST: INFO: check_command => check-bench-disk
+  mer. 28 mars 2018 14:27:35 CEST: INFO: check_period => 24x7
+  mer. 28 mars 2018 14:27:35 CEST: INFO: type => 65560
+  mer. 28 mars 2018 14:27:35 CEST: INFO: last_hard_state => 0
+
+.. note::
+   This log file will grow quickly, do not forget to add a log rotate
+
+Use parameters
+**************
+
+The Centreon Broker log function should be use for log only. To write into a file, 
+we must use the LUA dedicated function. Moreover, it is possible to use parameters 
+to define the name of the log file.
+
+Edit your LUA connector:
+
+.. code-block:: lua
+
+  function init(conf)
+    logFile = conf['logFile']
+    broker_log:set_parameters(3, "/var/log/centreon-broker/debug.log")
+  end
+  
+  function write(d)
+    for k,v in pairs(d) do
+      wrintIntoFile(tostring(k .. " => " .. tostring(v)).. "\n")
+    end
+    return true
+  end
+
+  function wrintIntoFile(output)
+    local file,err = io.open(logFile, 'a')
+    if file == nil then
+      broker_log:info(3, "Couldn't open file: "..err)
+    else
+      file:write(output)
+      file:close()
+    end
+  end
+
+The **init()** function allows to get parameters and define these from Centreon 
+web interface. See technical documentation for more information.
+
+Edit also your broker output to declare this parameter:
+
+.. image:: /_static/images/developper/lua/add_parameter.png
+   :align: center
+
+Then click **Save** and go to generate the configuration and restart **cbd**.
+
+.. note::
+   Don’t forget to restart “centengine” too to create the Centreon Broker cache.
```

Comment:
I don't know if it is useful to talk about this here. You don't use the cache here.
Maybe it will be better to talk about it when you will use it...